common/config/rush/pnpm-lock.yaml

```diff
@@ -8142,28 +8131,6 @@ packages:
       redux: 4.0.5
     dev: false
 
-  /react-redux/7.2.2_react@17.0.2+redux@4.0.5:
```

Comment:
Notice that this second copy of `react-redux` is being removed.

Reply:
will try that this weekend. Thanks @iclanton